### PR TITLE
Added ARM64 jobs to .travis.yml file and Updated the STATICCHECK_VERS…

### PR DESCRIPTION
**Pull request by vishal-biyani**
…ION to support arm64

Signed-off-by: odidev <odidev@puresoftware.com>

For integration test of https://github.com/fission/fission/pull/1689

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1705)

### CODE REVIEW

File: hack/travis-kube-setup.sh [hunk 1]
```diff
@@ -33,9 +41,9 @@ fi
 HELM_VERSION=2.14.0
```
Comment (harshthakur9030) [inline hack/travis-kube-setup.sh:41]: Can we use helm 3?

File: pkg/storagesvc/client/storagesvc_test.go [hunk 2]
```diff
@@ -57,10 +58,18 @@ func MakeTestFile(size int) *os.File {
 	return f
 }
 
+func checkArch() string {
+	if runtime.GOARCH == "arm64" {
+		return "RELEASE.2020-07-24T22-43-05Z-arm64"
```
Comment (harshthakur9030) [inline pkg/storagesvc/client/storagesvc_test.go:63]: Can we use git commit / git commit + date to track? 
Something like [this](https://github.com/openebs/node-disk-manager/blob/master/build/build.sh#L20)?

**Comment by harshthakur9030** [comment]
Can we close https://github.com/fission/fission/pull/1689 ? Looks very similar.

**Comment by vishal-biyani** [comment]
@harshthakur9030 I actually created this from 1689 to run integration tests.

**Comment by RealHarshThakur** [comment]
Closing it  as  we've moved to Github Actions in #1876 and most of the other checks are fixed/being fixed by Golang CI lint PRs .

**Pull request closed without merging**